internal/slackerror/errors.go

```diff
@@ -711,6 +713,18 @@ Otherwise start your app for local development with: %s`,
 		Message: "Failed to get teams for restricted user",
 	},
 
+	ErrFeedbackNameInvalid: {
+		Code:        ErrFeedbackNameInvalid,
+		Message:     "Invalid feedback name provided",
+		Remediation: fmt.Sprintf("View the feedback options with %s", style.Commandf("feedback --help", false)),
+	},
+
+	ErrFeedbackNameRequired: {
+		Code:        ErrFeedbackNameRequired,
+		Message:     "Please provide a feedback --name flag or remove the --no-prompt flag",
+		Remediation: fmt.Sprintf("View feedback options with %s", style.Commandf("feedback --help", false)),
+	},
+
```

Comment:
note: The new `slackerror` codes that have been added. I also clarified the `ErrFeedbackNameRequired` message with `--name flag` .